subprojects/libero-distribution/README.md

```diff
@@ -97,8 +149,14 @@ I Release You!
      destination:
        remote: "origin"
        branch: "master"
+     message:
+       prefix: "[release]"
    maven:
-     repo: "releases:default:http://nexus.fake.site/nexus/content/repositories/releases/"
+     repo:
+       local: "/path/to/local/repo"
+       remote: "releases::default::http://nexus.fake.site/nexus/content/repositories/releases/"
+       clean: "foo.bar,baz.qux"
```

Comment:
Yeah probably would be better there, but I'm also going to add a ticket to go back through and clean up/simplify the configuration model. That might warrant a larger discussion :-D